--- a/Server/DataStructures/InternalDataFacade.h
+++ b/Server/DataStructures/InternalDataFacade.h
@@ -210,7 +210,7 @@
 
         unsigned number_of_chars = 0;
         name_stream.read((char *)&number_of_chars, sizeof(unsigned));
-        BOOST_ASSERT_MSG(0 != number_of_chars, "name file broken");
+	//BOOST_ASSERT_MSG(0 != number_of_chars, "name file broken");
         m_names_char_list.resize(number_of_chars + 1); //+1 gives sentinel element
         name_stream.read((char *)&m_names_char_list[0], number_of_chars * sizeof(char));
         if (0 == m_names_char_list.size())
